addon/-private/system/references/has-many.js

```diff
@@ -147,6 +147,7 @@ HasManyReference.prototype.link = function() {
    @return {Array} The ids in this has-many relationship
 */
 HasManyReference.prototype.ids = function() {
+  this.hasManyRelationship.manyArray;
```

Comment:
just curious what the effect of this is? is that what forces it to recognize the related update?

Reply:
`manyArray` is lazily created on access, by accessing it, `others.[]` now points at an array instead of undefined.

Reply:
ok I see nice